server/src/test/java/io/crate/auth/HttpAuthUpstreamHandlerTest.java

```diff
@@ -64,6 +64,11 @@
 
 public class HttpAuthUpstreamHandlerTest extends ESTestCase {
 
+    private final Settings globalJwtEnabled = Settings.builder()
+        .put("auth.host_based.enabled", true)
+        .put("auth.host_based.jwt.iss", "https:example.com")
+        .build();
+
```

Comment:
Can be removed? (probably all changes here can be reverted)

Reply:
👍 yep, forgot to clean it up